### DISCUSSION TITLE
v3.11.3

### DISCUSSION THREAD

**Discussion by hairyhenderson** [hairyhenderson/gomplate#1502] [General]
This is a very minor patch release to update a dependency that's causing gomplate to trigger vulnerability scanners for CVE-2022-27664.

To the best of my understanding gomplate is not vulnerable, as it doesn't expose any HTTP servers - this patch release is purely to help vulnerability scans pass!

## What's Changed
* Bump dependencies with unrelated CVEs by @hairyhenderson in https://github.com/hairyhenderson/gomplate/pull/1501

**Full Changelog**: https://github.com/hairyhenderson/gomplate/compare/v3.11.2...v3.11.3

<hr /><em>This discussion was created from the release <a href='https://github.com/hairyhenderson/gomplate/releases/tag/v3.11.3'>v3.11.3</a>.</em>

**Comment by mayurwaghmode** [comment]
@hairyhenderson looks like the [v3.11.3](https://github.com/hairyhenderson/gomplate/releases/tag/v3.11.3) release is not containing the ppc64le-specific binary. Is it expected? We included ppc64le binary in the [v3.11.2](https://github.com/hairyhenderson/gomplate/releases/tag/v3.11.2) release.

**Reply by hairyhenderson** [reply]
Ah - thanks @mayurwaghmode. I will build the ppc64le binary for that. For 3.11.2 I had to do that manually as that code is in the `main` branch, not in the `3.11.x` one.

I'll try to get the binary out soon. For now using 3.11.2 should be fine (other than the annoying scanner hits!)

**Reply by mayurwaghmode** [reply]
Thanks @hairyhenderson :)

**Reply by hairyhenderson** [reply]
The new binary for ppc64le is up at https://github.com/hairyhenderson/gomplate/releases/download/v3.11.3/gomplate_linux-ppc64le
